src/main/java/Freedom.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+
+public class Freedom {
+    protected static Task[] storage = new Task[100];
+    static int lastIndex = 0;
+
+    public static void main(String[] args) {
+        String logo = "\t_________________________________________\n";
+        String message = """
+                \tHello! I'm Freedom
+                \tWhat can I do for you?
+                """;
+        String closing = "\tBye. Hope to see you again soon!\n";
+
+        System.out.println(logo + message + logo);
+
+        while (true) {
+            Scanner in = new Scanner(System.in);
+            String line = in.nextLine();
+
+            if (line.equals("bye")) {
+                break;
+            }
+            boolean isCommand = handleInput(line);
+            if (!isCommand) {
+                lastIndex++;
+            }
+        }
+
+        System.out.println(logo + closing + logo);
+    }
+
+    public static boolean handleInput(String input) {
+        String[] words = input.split(" ");
+        String description;
+
+        if (input.equals("list")) {
+            printList();
+            return true;
+        }
+        if (words[0].equals("mark")) {
+            int listNumber = Integer.parseInt(words[1]);
+            storage[listNumber - 1].markDone();
+            markTaskStatement(storage[listNumber - 1], true);
+            return true;
+        }
+        if (words[0].equals("unmark")) {
+            int listNumber = Integer.parseInt(words[1]);
+            storage[listNumber - 1].markUndone();
+            markTaskStatement(storage[listNumber - 1], false);
+            return true;
+        }
+        if (words[0].equals("todo")) {
+            description = input.replaceFirst("todo", "");
+            storage[lastIndex] = new ToDo(description);
+        }
+        else if (words[0].equals("deadline")) {
+            description = input.replaceFirst("deadline", "");
+            String[] members = description.split("/");
+            storage[lastIndex] = new Deadline(members[0], members[1]);
```

Comment:
Perhaps it would be better to use a switch statement instead of using a long set if else statements for clarity.
Additionally the code standard is to use:
if {
.....
} else {
....
}

for if else statements